src/odemis/util/filename.py

```diff
@@ -198,6 +198,27 @@ def generate_fn(ptn, count):
 
     return os.path.join(path, fn + ext)
 
+def individualize_filename(fn, names):
+    """
+    Creates a filename based on the input filename which is unique in a list of filenames by adding a counter.
+    input
+        fn (str): proposed filename
```

Comment:
Is that really a filename, or just a name? If it's a filename, maybe the function should take care of inserting the counter before the extension?